### PR TITLE
Fix charting menu character icons when switching player1 or player2 + Proper week 4 hair blowing animation

### PR DESCRIPTION
**Pull request by Spel0**
Right now it's just stuck on previously selected character until you save and reload JSON

### CODE REVIEW

**Comment by Spel0** [comment]
Kade, we're on KE 1.7 and you **still** don't have proper hair blowing animation in week 4 from week 7 update, like step up your game, do I need to do everything for you? 💀

File: assets/shared/images/characters/bf-carOffsets.txt [hunk 1]
```diff
@@ -1,4 +1,5 @@
 idle -5 0
+idleHair -5 0
```
Comment (RagdollClash) [inline assets/shared/images/characters/bf-carOffsets.txt:2]: you're missing the idleHair offset for mom-car

**Review by RagdollClash — CHANGES_REQUESTED** [review]

File: assets/shared/images/characters/bf-carOffsets.txt (hunk 1 above)
Comment (Spel0) [reply assets/shared/images/characters/bf-carOffsets.txt:2]: > you're missing the idleHair offset for mom-car

mom-car doesn't need it as she have 0 0 idle and not -5 0 like bf. Plus it still puts her into idleHair animation in offset debug menu after her normal idle animation is finished

Comment (RagdollClash) [reply assets/shared/images/characters/bf-carOffsets.txt:2]: ahh

**Pull request merged**